python_files/supplementary_plot_helpers.py

```diff
@@ -19,6 +20,13 @@
     11, 12, 13, 14, 15, 17, 18, 20, 22, 23, 24, 28, 29, 30, 31, 32, 33, 34, 35,
     36, 39, 40, 41, 42, 43, 44, 45, 46, 47
 ]
+TIMEPOINT_INDICES = ["primary", "baseline", "post_induction", "on_nivo"]
+TIMEPOINT_PAIRS = [
```

Comment:
What was the logic for making these global variables within the helpers file?

Reply:
This was mostly for convenience, before adding the `baseline to on_nivo` pair I had originally used `itertools` to compute each individual pair. Agreed this doesn't serve as much of a purpose anymore, I can leave them hard-coded inside the function instead.